lib/phabricator/project.rb

```diff
@@ -10,7 +10,7 @@ class Project
     def self.populate_all
       response = client.request(:post, 'project.query')
 
-      response['result'].each do |phid, data|
+      response['result']['data'].each do |phid, data|
```

Comment:
maybe phabricator changed the structure of the json? It's definitely sending the data nested inside a `data` key.